deploy-wazuh-amzn2023-docker-host

```diff
@@ -105,10 +105,13 @@ yum -y remove python3-requests
 yum -y install python3-pip
 /usr/bin/pip3 install docker urllib3 requests
 
-# Install cron (cronie) which is no included by default in Amazon Linux 2023
-yum install cronie -y
+# Install cron (cronie) and rsyslog, which are no included by default in Amazon Linux 2023
+yum -y install cronie 
 systemctl enable crond.service
 systemctl start crond.service
+yum -t install rsyslog 
```

Comment:
I'm waiting on confirmation that this `-t` is correct. I suspect it was meant to be `-y`.